tests/hap/test_svm_demo.py

```diff
@@ -0,0 +1,146 @@
+""" This module tests full pipeline SVM processing - visit has one detector and uses one filter.
+
+"""
+import datetime
+import glob
+import os
+import pytest
+
+from drizzlepac.haputils import astroquery_utils as aqutils
+from drizzlepac import runsinglehap
+from astropy.io import fits, ascii
+from pathlib import Path
+
+"""
+    test_svm_demo.py
+
+    This test file can be executed in the following manner:
+        $ pytest -s --basetemp=/internal/hladata/yourUniqueDirectoryHere test_svm.py >& test_svm.log &
+        $ tail -f test_svm.log
+      * Note: When running this test, the `--basetemp` directory should be set to a unique
+        existing directory to avoid deleting previous test output.
+      * The POLLER_FILE exists in the tests/hap directory.
+
+    *** The --basetemp does not seem to be working at this time!!!
+
+"""
+
+WCS_SUB_NAME = "FIT_SVM_GAIA"
+POLLER_FILE = "acs_8f6_61_input.out"
```

Comment:
This effectively hard-codes the test for this single dataset.  Why not assign this poller file a more generic name, since the code itself doesn't really care, then the test can be updated later by simply replacing the contents without having to change this test as well?  For example, rename the current file as 'svm_test1_input.out', then use that as "POLLER_FILE".  What is here will still work, but is not any different from simply having the contents of the file in this module directly with the way it is defined.

Reply:
This is true.  Since I mean for this code to be a demonstration and an example for discussion before I parameterize or do other complex stuff, I thought this was OK for now.  I see there are JWST examples where they have done this hard-coding for testing of particular results.  Passing parameters to fixtures is awkward (as I have been researching do this very thing).

Reply:
Good for this test then.